fix topic name for camera teleop parameter

related to https://github.com/pollen-robotics/gstreamer_webrtc/issues/53

when I try to launch the core on beta2, Rviz is launching then it crashes with this error : 
` 
[pollen_kdl_kinematics-21] Traceback (most recent call last):
[pollen_kdl_kinematics-21]   File "/home/reachy/reachy_ws/install/pollen_kdl_kinematics/lib/pollen_kdl_kinematics/pollen_kdl_kinematics", line 33, in <module>
[pollen_kdl_kinematics-21]     sys.exit(load_entry_point('pollen-kdl-kinematics', 'console_scripts', 'pollen_kdl_kinematics')())
[pollen_kdl_kinematics-21]   File "/home/reachy/reachy_ws/build/pollen_kdl_kinematics/pollen_kdl_kinematics/pollen_kdl_kinematics_node.py", line 727, in main
[pollen_kdl_kinematics-21]     node = PollenKdlKinematics()
[pollen_kdl_kinematics-21]   File "/home/reachy/reachy_ws/build/pollen_kdl_kinematics/pollen_kdl_kinematics/pollen_kdl_kinematics_node.py", line 197, in __init__
[pollen_kdl_kinematics-21]     chain, fk_solver, ik_solver = generate_solver(
[pollen_kdl_kinematics-21]   File "/home/reachy/reachy_ws/build/pollen_kdl_kinematics/pollen_kdl_kinematics/kdl_kinematics.py", line 14, in generate_solver
[pollen_kdl_kinematics-21]     success, urdf_tree = urdf.treeFromString(urdf_str)
[pollen_kdl_kinematics-21]   File "/home/reachy/reachy_ws/build/pollen_kdl_kinematics/pollen_kdl_kinematics/kdl_parser_py/urdf.py", line 31, in treeFromString
[pollen_kdl_kinematics-21]     return treeFromUrdfModel(urdf.URDF.from_xml_string(xml))
[pollen_kdl_kinematics-21]   File "/home/reachy/reachy_ws/build/pollen_kdl_kinematics/pollen_kdl_kinematics/kdl_parser_py/urdf.py", line 123, in treeFromUrdfModel
[pollen_kdl_kinematics-21]     if not _add_children_to_tree(robot_model, robot_model.link_map[child], tree):
[pollen_kdl_kinematics-21]   File "/home/reachy/reachy_ws/build/pollen_kdl_kinematics/pollen_kdl_kinematics/kdl_parser_py/urdf.py", line 96, in _add_children_to_tree
[pollen_kdl_kinematics-21]     if not _add_children_to_tree(robot_model, child, tree):
[pollen_kdl_kinematics-21]   File "/home/reachy/reachy_ws/build/pollen_kdl_kinematics/pollen_kdl_kinematics/kdl_parser_py/urdf.py", line 76, in _add_children_to_tree
[pollen_kdl_kinematics-21]     inert = _toKdlInertia(root.inertial)
[pollen_kdl_kinematics-21] KeyboardInterrupt
[ros2_control_node-11] [INFO]: signal_handler(signum=2)
[dynamic_state_router-9] Traceback (most recent call last):
[dynamic_state_router-9]   File "/home/reachy/reachy_ws/install/dynamic_state_router/lib/dynamic_state_router/dynamic_state_router", line 33, in <module>
[dynamic_state_router-9]     sys.exit(load_entry_point('dynamic-state-router', 'console_scripts', 'dynamic_state_router')())
[dynamic_state_router-9]   File "/home/reachy/reachy_ws/build/dynamic_state_router/dynamic_state_router/dynamic_state_router.py", line 400, in main
[dynamic_state_router-9]     rclpy.spin(node)
[dynamic_state_router-9]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 222, in spin
[dynamic_state_router-9]     executor.spin_once()
[dynamic_state_router-9]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 739, in spin_once
[dynamic_state_router-9]     self._spin_once_impl(timeout_sec)
[dynamic_state_router-9]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 728, in _spin_once_impl
[dynamic_state_router-9]     handler, entity, node = self.wait_for_ready_callbacks(timeout_sec=timeout_sec)
[dynamic_state_router-9]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 711, in wait_for_ready_callbacks
[dynamic_state_router-9]     return next(self._cb_iter)
[dynamic_state_router-9]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 608, in _wait_for_ready_callbacks
[dynamic_state_router-9]     wait_set.wait(timeout_nsec)
[dynamic_state_router-9] KeyboardInterrupt
[ros2-10] [INFO]: Writing remaining messages from cache to the bag. It may take a while
[ros2-10] [INFO]: Event publisher thread: Exiting
[ros2-10] [INFO]: Recording stopped
[goto_server-8] Traceback (most recent call last):
[goto_server-8]   File "/home/reachy/reachy_ws/install/pollen_goto/lib/pollen_goto/goto_server", line 33, in <module>
[goto_server-8]     sys.exit(load_entry_point('pollen-goto', 'console_scripts', 'goto_server')())
[goto_server-8]   File "/home/reachy/reachy_ws/build/pollen_goto/pollen_goto/goto_action_server.py", line 420, in main
[goto_server-8]     rclpy.shutdown()
[goto_server-8]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 126, in shutdown
[goto_server-8]     _shutdown(context=context)
[goto_server-8]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/utilities.py", line 58, in shutdown
[goto_server-8]     return context.shutdown()
[goto_server-8]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/context.py", line 102, in shutdown
[goto_server-8]     self.__context.shutdown()
[goto_server-8] rclpy._rclpy_pybind11.RCLError: failed to shutdown: rcl_shutdown already called on the given context, at ./src/rcl/init.c:241
[ros2-10] [INFO]: Recording stopped
[component_container-7] [INFO]: signal_handler(signum=2)
[reachy_grpc_video_sdk_server-6] Traceback (most recent call last):
[reachy_grpc_video_sdk_server-6]   File "/home/reachy/reachy_ws/install/reachy_sdk_server/lib/reachy_sdk_server/reachy_grpc_video_sdk_server", line 33, in <module>
[reachy_grpc_video_sdk_server-6] Exception in thread Thread-1 (spin_ros):
[reachy_grpc_video_sdk_server-6] Traceback (most recent call last):
[reachy_grpc_video_sdk_server-6]   File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
[reachy_grpc_video_sdk_server-6]     sys.exit(load_entry_point('reachy-sdk-server', 'console_scripts', 'reachy_grpc_video_sdk_server')())
[reachy_grpc_video_sdk_server-6]   File "/home/reachy/reachy_ws/build/reachy_sdk_server/reachy_sdk_server/grpc_server/video_server.py", line 356, in main
[reachy_grpc_video_sdk_server-6]     self.run()
[reachy_grpc_video_sdk_server-6]   File "/usr/lib/python3.10/threading.py", line 953, in run
[reachy_grpc_video_sdk_server-6]     server.wait_for_termination()
[reachy_grpc_video_sdk_server-6]   File "/home/reachy/.local/lib/python3.10/site-packages/grpc/_server.py", line 1350, in wait_for_termination
[reachy_grpc_video_sdk_server-6]     self._target(*self._args, **self._kwargs)
[reachy_grpc_video_sdk_server-6]   File "/home/reachy/reachy_ws/build/reachy_sdk_server/reachy_sdk_server/grpc_server/video_server.py", line 179, in spin_ros
[reachy_grpc_video_sdk_server-6]     return _common.wait(
[reachy_grpc_video_sdk_server-6]   File "/home/reachy/.local/lib/python3.10/site-packages/grpc/_common.py", line 156, in wait
[reachy_grpc_video_sdk_server-6]     rclpy.spin(self.node)
[reachy_grpc_video_sdk_server-6]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 222, in spin
[reachy_grpc_video_sdk_server-6]     _wait_once(wait_fn, MAXIMUM_WAIT_TIMEOUT, spin_cb)
[reachy_grpc_video_sdk_server-6]   File "/home/reachy/.local/lib/python3.10/site-packages/grpc/_common.py", line 116, in _wait_once
[reachy_grpc_video_sdk_server-6]     executor.spin_once()
[reachy_grpc_video_sdk_server-6]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 739, in spin_once
[reachy_grpc_video_sdk_server-6]     wait_fn(timeout=timeout)
[reachy_grpc_video_sdk_server-6]   File "/usr/lib/python3.10/threading.py", line 607, in wait
[reachy_grpc_video_sdk_server-6]     self._spin_once_impl(timeout_sec)
[reachy_grpc_video_sdk_server-6]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 728, in _spin_once_impl
[reachy_grpc_video_sdk_server-6]     signaled = self._cond.wait(timeout)
[reachy_grpc_video_sdk_server-6]   File "/usr/lib/python3.10/threading.py", line 324, in wait
[reachy_grpc_video_sdk_server-6]     handler, entity, node = self.wait_for_ready_callbacks(timeout_sec=timeout_sec)
[reachy_grpc_video_sdk_server-6]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 711, in wait_for_ready_callbacks
[reachy_grpc_video_sdk_server-6]     gotit = waiter.acquire(True, timeout)
[reachy_grpc_video_sdk_server-6] KeyboardInterrupt
[gripper_safe_controller-5] Traceback (most recent call last):
[gripper_safe_controller-5]   File "/home/reachy/reachy_ws/install/gripper_safe_controller/lib/gripper_safe_controller/gripper_safe_controller", line 33, in <module>
[gripper_safe_controller-5]     sys.exit(load_entry_point('gripper-safe-controller', 'console_scripts', 'gripper_safe_controller')())
[gripper_safe_controller-5]   File "/home/reachy/reachy_ws/build/gripper_safe_controller/gripper_safe_controller/gripper_safe_controller.py", line 269, in main
[gripper_safe_controller-5]     rclpy.spin(gripper)
[gripper_safe_controller-5]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 222, in spin
[gripper_safe_controller-5]     executor.spin_once()
[gripper_safe_controller-5]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 739, in spin_once
[gripper_safe_controller-5]     self._spin_once_impl(timeout_sec)
[gripper_safe_controller-5]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 728, in _spin_once_impl
[gripper_safe_controller-5]     handler, entity, node = self.wait_for_ready_callbacks(timeout_sec=timeout_sec)
[gripper_safe_controller-5]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 711, in wait_for_ready_callbacks
[gripper_safe_controller-5]     return next(self._cb_iter)
[gripper_safe_controller-5]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 608, in _wait_for_ready_callbacks
[gripper_safe_controller-5]     wait_set.wait(timeout_nsec)
[gripper_safe_controller-5] KeyboardInterrupt
[robot_state_publisher-3] [INFO]: signal_handler(signum=2)
[hal-2] [WARN]: Emergency shutdown!
[hal-2] Failed to publish log message to rosout: publisher's context is invalid, at ./src/rcl/publisher.c:389
[pollen_kdl_kinematics-21] ****************************************
[pollen_kdl_kinematics-21] pollen_kdl_kinematics_node::python_gc stop: 61.462ms
[pollen_kdl_kinematics-21] ****************************************
[ERROR] [goto_server-8]: process has died [pid 2316, exit code 1, cmd '/home/reachy/reachy_ws/install/pollen_goto/lib/pollen_goto/goto_server --ros-args'].
[INFO] [launch.user]: Critical node failed : [goto_server]
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Cannot shutdown a ROS adapter that is not running
[component_container-7] [INFO]: Color frame thread exit!
[ERROR] [reachy_grpc_video_sdk_server-6]: process has died [pid 2312, exit code -2, cmd '/home/reachy/reachy_ws/install/reachy_sdk_server/lib/reachy_sdk_server/reachy_grpc_video_sdk_server --ros-args'].
[INFO] [launch.user]: Critical node failed : [reachy_grpc_video_sdk_server]
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Cannot shutdown a ROS adapter that is not running
[ERROR] [robot_state_publisher-3]: process has died [pid 2306, exit code -11, cmd '/opt/ros/humble/lib/robot_state_publisher/robot_state_publisher --ros-args --params-file /tmp/launch_params_m3n_nsrn'].
[INFO] [launch.user]: Critical node failed : [robot_state_publisher]
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Cannot shutdown a ROS adapter that is not running
[rplidar_scan_publisher-1] [INFO]: Stop motor
[ERROR] [dynamic_state_router-9]: process has died [pid 2318, exit code -2, cmd '/home/reachy/reachy_ws/install/dynamic_state_router/lib/dynamic_state_router/dynamic_state_router /home/reachy/reachy_ws/install/reachy_bringup/share/reachy_bringup/config/reachy_full_kit_controllers.yaml --ros-args'].
[INFO] [launch.user]: Critical node failed : [dynamic_state_router]
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Cannot shutdown a ROS adapter that is not running
[ERROR] [gripper_safe_controller-5]: process has died [pid 2310, exit code -2, cmd '/home/reachy/reachy_ws/install/gripper_safe_controller/lib/gripper_safe_controller/gripper_safe_controller --controllers-file /home/reachy/reachy_ws/install/reachy_bringup/share/reachy_bringup/config/reachy_full_kit_controllers.yaml --ros-args'].
[INFO] [launch.user]: Critical node failed : [gripper_safe_controller]
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Cannot shutdown a ROS adapter that is not running
[INFO] [ros2-10]: process has finished cleanly [pid 2320]
[INFO] [rviz2-22]: process has finished cleanly [pid 2657]
[INFO] [rplidar_scan_publisher-1]: process has finished cleanly [pid 2302]
[ERROR] [pollen_kdl_kinematics-21]: process has died [pid 2655, exit code -2, cmd '/home/reachy/reachy_ws/install/pollen_kdl_kinematics/lib/pollen_kdl_kinematics/pollen_kdl_kinematics --ros-args -r __node:=kinematics -r __ns:=/'].
[INFO] [launch.user]: Critical node failed : [pollen_kdl_kinematics]
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Cannot shutdown a ROS adapter that is not running
[hal-2] Traceback (most recent call last):
[hal-2]   File "/home/reachy/reachy_ws/build/zuuu_hal/zuuu_hal/zuuu_hal.py", line 1489, in main
[hal-2]     rclpy.spin(node)
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 222, in spin
[hal-2]     executor.spin_once()
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 739, in spin_once
[hal-2]     self._spin_once_impl(timeout_sec)
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 734, in _spin_once_impl
[hal-2]     handler()
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/task.py", line 239, in __call__
[hal-2]     self._handler.send(None)
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 437, in handler
[hal-2]     await call_coroutine(entity, arg)
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 351, in _execute_timer
[hal-2]     await await_or_execute(tmr.callback)
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 107, in await_or_execute
[hal-2]     return callback(*args)
[hal-2]   File "/home/reachy/reachy_ws/build/zuuu_hal/zuuu_hal/zuuu_hal.py", line 1452, in main_tick
[hal-2]     self.read_measurements()
[hal-2]   File "/home/reachy/reachy_ws/build/zuuu_hal/zuuu_hal/zuuu_hal.py", line 1079, in read_measurements
[hal-2]     self.omnibase.read_all_measurements()
[hal-2]   File "/home/reachy/reachy_ws/build/zuuu_hal/zuuu_hal/zuuu_hal.py", line 158, in read_all_measurements
[hal-2]     self.back_wheel_measurements = self.back_wheel.get_measurements()
[hal-2]   File "/home/reachy/dev/PyVESC/src/pyvesc/VESC/VESC.py", line 151, in get_measurements
[hal-2]     return self.write(self._get_values_msg, num_read_bytes=self._get_values_msg_expected_length)
[hal-2]   File "/home/reachy/dev/PyVESC/src/pyvesc/VESC/VESC.py", line 110, in write
[hal-2]     response, consumed = decode(self.serial_port.read(num_read_bytes + 6))
[hal-2]   File "/home/reachy/.local/lib/python3.10/site-packages/serial/serialposix.py", line 565, in read
[hal-2]     ready, _, _ = select.select([self.fd, self.pipe_abort_read_r], [], [], timeout.time_left())
[hal-2] KeyboardInterrupt
[hal-2] 
[hal-2] During handling of the above exception, another exception occurred:
[hal-2] 
[hal-2] Traceback (most recent call last):
[hal-2]   File "/home/reachy/reachy_ws/install/zuuu_hal/lib/zuuu_hal/hal", line 33, in <module>
[hal-2]     sys.exit(load_entry_point('zuuu-hal', 'console_scripts', 'hal')())
[hal-2]   File "/home/reachy/reachy_ws/build/zuuu_hal/zuuu_hal/zuuu_hal.py", line 1495, in main
[hal-2]     rclpy.shutdown()
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 126, in shutdown
[hal-2]     _shutdown(context=context)
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/utilities.py", line 58, in shutdown
[hal-2]     return context.shutdown()
[hal-2]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/context.py", line 102, in shutdown
[hal-2]     self.__context.shutdown()
[hal-2] rclpy._rclpy_pybind11.RCLError: failed to shutdown: rcl_shutdown already called on the given context, at ./src/rcl/init.c:241
[INFO] [ros2_control_node-11]: process has finished cleanly [pid 2496]
[component_container-7] [WARN]: Do destroy ~OBCameraNode
[component_container-7] [WARN]: stop streams
[ERROR] [component_container-7]: process[component_container-7] failed to terminate '5' seconds after receiving 'SIGINT', escalating to 'SIGTERM'
[ERROR] [hal-2]: process[hal-2] failed to terminate '5' seconds after receiving 'SIGINT', escalating to 'SIGTERM'
[INFO] [component_container-7]: sending signal 'SIGTERM' to process[component_container-7]
[INFO] [hal-2]: sending signal 'SIGTERM' to process[hal-2]
[component_container-7] [INFO]: signal_handler(signum=15)
[component_container-7] [WARN]: Destroy ~OBCameraNode DONE
[component_container-7] strmh->payload_header_hold_bytes is invalid,size:0
[component_container-7] strmh->payload_header_hold_bytes is invalid,size:0
[INFO] [component_container-7]: process has finished cleanly [pid 2314]
[ERROR] [hal-2]: process[hal-2] failed to terminate '10.0' seconds after receiving 'SIGTERM', escalating to 'SIGKILL'
[INFO] [hal-2]: sending signal 'SIGKILL' to process[hal-2]
[ERROR] [hal-2]: process has died [pid 2304, exit code -9, cmd '/home/reachy/reachy_ws/install/zuuu_hal/lib/zuuu_hal/hal --ros-args -r __node:=zuuu_hal --params-file /home/reachy/reachy_ws/install/zuuu_hal/share/zuuu_hal/config/params.yaml --params-file /tmp/launch_params_62thuh5r'].
`

--- a/reachy_sdk_server/reachy_sdk_server/grpc_server/reachy.py
+++ b/reachy_sdk_server/reachy_sdk_server/grpc_server/reachy.py
@@ -19,12 +19,7 @@ from reachy2_sdk_api.reachy_pb2 import (
 from reachy2_sdk_api.reachy_pb2_grpc import add_ReachyServiceServicer_to_server
 
 from ..abstract_bridge_node import AbstractBridgeNode
-from ..utils import (
-    endless_timer_get_stream,
-    endless_timer_get_stream_works,
-    get_current_timestamp,
-    parse_reachy_config,
-)
+from ..utils import endless_timer_get_stream, endless_timer_get_stream_works, get_current_timestamp, parse_reachy_config
 from .arm import ArmServicer
 from .hand import HandServicer
 from .head import HeadServicer

--- a/reachy_sdk_server/reachy_sdk_server/grpc_server/video_server.py
+++ b/reachy_sdk_server/reachy_sdk_server/grpc_server/video_server.py
@@ -128,14 +128,14 @@ class ReachyGRPCVideoSDKServicer:
 
         self._left_camera_info_sub = self.node.create_subscription(
             CameraInfo,
-            "teleop_camera/left_image/camera_info",
+            "teleop_camera/left_image/image_raw/camera_info",
             partial(self.on_info_update, cam_type=CameraType.TELEOP, side=View.LEFT),
             1,
         )
 
         self._right_camera_info_sub = self.node.create_subscription(
             CameraInfo,
-            "teleop_camera/right_image/camera_info",
+            "teleop_camera/right_image/image_raw/camera_info",
             partial(self.on_info_update, cam_type=CameraType.TELEOP, side=View.RIGHT),
             1,
         )

